### PR TITLE
Build the second page of Request flow

### PR DESCRIPTION
**Pull request by lupinthethirdgentleman**
Built the second page of Request flow
Built the third page of Request flow
Implemented whole interaction for Request flow
Screenshot_3
Screenshot_1
Screenshot_2

### CODE REVIEW

File: src/screens/RequestFlow/exchange.png
Comment (obasilakis) [inline src/screens/RequestFlow/exchange.png:1]: this is supposed to be an interactive button. Please remove

File: src/screens/RequestFlow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  };
+
+  const onContinue = () => {
+    console.log('onContinue');
+  };
+
+  return (
+    <SafeAreaView style={backgroundStyle}>
+      <View style={styles.viewStyle}>
+        <TouchableOpacity
```
Comment (obasilakis) [inline src/screens/RequestFlow/index.tsx:45]: the top screen should be part of the navigation and handled by the phone

File: src/screens/RequestFlow/pen.png
Comment (obasilakis) [inline src/screens/RequestFlow/pen.png:1]: please export svgs from figma

File: src/screens/RequestFlow/pen_white.png
Comment (obasilakis) [inline src/screens/RequestFlow/pen_white.png:1]: same as above

File: src/screens/RequestFlow/index.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+              {
+                ...fontStyle.fontH,
+              },
+            ]}
+          >
+            USD
+          </Text>
+        </View>
+        <View style={styles.quai} />
+        <View style={styles.quaiView}>
```
Comment (obasilakis) [inline src/screens/RequestFlow/index.tsx:99]: desired functionality here: when the user taps the exchange button, we swap the currencies. meaning QUAI amount goes to the input and USD amount goes under the input. the button text also changes to USD

File: src/screens/RequestFlow/exchange.png
Comment (lupinthethirdgentleman) [reply src/screens/RequestFlow/exchange.png:1]: I got it, Just updated it

File: src/screens/RequestFlow/index.tsx (hunk 2 above)
Comment (lupinthethirdgentleman) [reply src/screens/RequestFlow/index.tsx]: I fixed it with your words.

**Pull request closed without merging**